.github/workflows/commit.yml

```diff
@@ -22,7 +22,6 @@ jobs:
         CF_SERVICE_AUTH: ${{secrets.CF_SERVICE_AUTH}}
         HARVEST_RUNNER_APP_GUID: ${{secrets.HARVEST_RUNNER_APP_GUID}}
         FLASK_APP_SECRET_KEY: ${{secrets.FLASK_APP_SECRET_KEY}}
-        OPENID_PRIVATE_KEY: ${{secrets.OPENID_PRIVATE_KEY}}
```

Comment:
Removing this proactively as we don't need it for testing and it's not stored in GH as a secret.